src/core/dropdownmenu/dropdownmenu.component.ts

```diff
@@ -0,0 +1,42 @@
+import {CommonModule} from "@angular/common";
+import {Component, EventEmitter, Input, Output} from "@angular/core";
+
+@Component({
+  selector: "wisdom-dropdownmenu",
+  imports: [CommonModule],
+  templateUrl: "./dropdownmenu.component.html",
+  styles: ``,
+})
+export class DropdownmenuComponent {
+  /**
+   * module name, when no name applied -> Test
+   */
+  @Input() menuName: string = "Test";
+
+  /**
+   * default options if no array is submitted.
+   */
+  @Input() options: string[] = ["A", "B", "C"];
```

Comment:
For translations the input type would be better to be a `Record<string, string>`, with the key as selected choice and the value being a translatable string.